### PR TITLE
Update gitignore and renames

### PR DESCRIPTION
**Pull request by Matthias1590**
- Changed the .gitignore file to the appropriate version (Visual Studio project instead of a Unity project).
- Renamed the solution & project.

### CODE REVIEW

File: LethalAPI.Monsters/LethalAPI.Monsters.csproj
Comment (MaxWasUnavailable) [inline LethalAPI.Monsters/LethalAPI.Monsters.csproj:1]: This uses a different format from Core. Needs to be reworked to use the original sln + csproj format.

File: LethalAPI.Monsters/Properties/AssemblyInfo.cs
Comment (MaxWasUnavailable) [inline LethalAPI.Monsters/Properties/AssemblyInfo.cs:1]: This uses a different format from Core. Needs to be reworked to use the original sln + csproj format.

**Review by MaxWasUnavailable — CHANGES_REQUESTED** [review]
This uses a different format from Core. Needs to be reworked to use the original sln + csproj format.

**Comment by Redforce04** [comment]
Dang I just saw this after I made the same changes locally. Oh well

**Pull request closed without merging**